### DISCUSSION TITLE
Server working fine without CORS. To add it or not to add it?

### DISCUSSION THREAD

**Discussion by dreinon** [fastapi/fastapi#4133] [Questions] [answered]
I'm not using CORS and I have been able to send requests to my server successfully from my website.
Should I still add it?
My server is deployed in Google Cloud Run and has Bearer Auth.
Thanks!

**Comment by simondale00** [comment, marked as answer]
Hey @dreinon. I think it'll depend on your use-case(s).

If you intend for your server to be accessible from any other domain, scheme, or port, you should definitely add CORS. If this use-case isn't on your horizon, you probably don't need to bother with CORS support right now.

Best,
Simon
